domains/Links/Database/Migrations/2020_10_10_152640_alter_timestamps_columns_to_links_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterTimestampsColumnsToLinksTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('links', function (Blueprint $table) {
+            $table->dropColumn(["created_at", "updated_at", "approved_at", "deleted_at"]);
```

Comment:
In a continuous integration mindset, we never drop columns that might be in use, because that leads to loose of data. That's a valid strategy for unreleased software, which is not the case of this API. 

Notice we already have some releases, and although we might assume that doing this is very unlikely to break anyone's code, we need to always think about what if someone already has data inserted that they don't want to lose?

I suggest we change this to use the `change()` method and let the database engine execute the proper SQL to update the already existent columns to the new type.

Reply:
@josepostiga sorry for the delay answering this. Since the documentation, states that the ["Only the following column types can be changed"](https://laravel.com/docs/8.x/migrations), I've decided to not use the timestampsTz method and instead used the datetimeTz method, which allows me to change the types of the column!

Hope to contribute more! 😄